test/selenium_tests/test_workflow_run.py

```diff
@@ -211,8 +211,10 @@ def _set_num_lines_to_3(self, tour_id):
         # for random_lines num_lines parameter as runtime parameter in workflow form.
         div = self.tool_parameter_div(tour_id)
         input_element = div.find_element_by_css_selector("input")
-        initial_value = input_element.get_attribute("value")
-        assert initial_value == "1", initial_value
```

Comment:
Alternatively we could just test for the initial_value to be "" instead of 1.

Reply:
@guerler That also seemed a good idea, are you going to open a PR for that?

Reply:
This would break ``test_subworkflow_runtime_parameters`` presumably, though is that an indication the other PR should somehow be applied to subworkflows also?

Reply:
Right, those two test cases would have to be distinguished. The latter depends on how subworkflows currently process runtime values. This is a good point.